client/src/templates/Introduction/components/block.tsx

```diff
@@ -118,21 +113,15 @@ class Block extends Component<BlockProps> {
     });
 
     const isProjectBlock = challenges.some(({ challenge }) => {
-      const isTakeHomeProject = blockDashedName === 'take-home-projects';
-
-      const projectCondition = [
-        isFinalProject(challenge.challengeType),
-        isCodeAllyPractice(challenge.challengeType)
-      ].some(Boolean);
-
-      return projectCondition && !isTakeHomeProject;
+      return (
+        isProjectBased(challenge.challengeType) &&
+        blockDashedName !== 'take-home-projects'
```

Comment:
as is this

Reply:
Yep, special cases suck. Passing both challenge type and blockDashedName as args seems fine for now, though.